### PR TITLE
updated changes for Josh

### PR DESCRIPTION
**Pull request by Josh-Woodcock**

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -126,41 +126,30 @@
...
-            "Data visualization",
-            "Plotly",
-            "Interaction design",
             "Linux",
             "Microsoft Windows",
             "macOS",
             "Version Control",
-            "Git"
+            "Git",
+            "GitHub Projects",
```
Comment (aharwood2) [inline people.json:145]: I don't think this is a Wikipedia main page heading so won't work

**Review by aharwood2 — CHANGES_REQUESTED** [review]
Could you check the vocabulary for the highlighted item?

File: people.json (hunk 1 above)
Comment (Josh-Woodcock) [reply people.json]: This has been resolved in latest commit

**Review by Josh-Woodcock — COMMENTED** [review]
Resolved with latest commit

**Review by ianhinder — APPROVED** [review]

**Pull request merged**